backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/BaseImagesCommand.java

```diff
@@ -298,6 +299,10 @@ protected void addDiskImageToDb(DiskImage image, CompensationContext compensatio
             compensationContext.snapshotNewEntity(imageStorageDomainMap);
             if (isDiskAdded) {
                 compensationContext.snapshotNewEntity(image);
+            } else {
+                BaseDisk baseDisk = baseDiskDao.get(image.getId());
+                compensationContext.snapshotEntityUpdated(baseDisk);
+                baseDiskDao.update(baseDisk);
```

Comment:
what's the purpose of this part?

Reply:
I think it's a leftover from the previous implementation where the base disk needed to be updated with next sequence number, I'll remove it and see what happens